frontend/src/static/js/components/webstatus-overview-table.ts

```diff
@@ -113,6 +117,44 @@ export class WebstatusOverviewTable extends LitElement {
     ];
   }
 
+  findFeatureFromAtom(
+    searchKey: string,
+    SearchValue: string,
+  ): components['schemas']['Feature'] | undefined {
+    if (!this.taskTracker.data?.data) {
+      return undefined;
+    }
+
+    for (const feature of this.taskTracker.data.data) {
+      if (searchKey === 'id' && feature?.feature_id === SearchValue) {
+        return feature;
+      } else if (
+        searchKey === 'name' &&
+        (feature?.feature_id.includes(SearchValue) ||
+          feature?.name.includes(SearchValue))
+      ) {
+        return feature;
+      }
+    }
+    return undefined;
+  }
+
+  sortDataOrder(): components['schemas']['Feature'][] | undefined {
+    if (!this.bookmark || !this.bookmark.is_ordered) {
+      return undefined;
+    }
+    const atoms: string[] = this.bookmark.query.trim().split('OR');
+    const features = [];
+    for (const atom of atoms) {
+      const terms = atom.trim().split(':');
+      const foundFeature = this.findFeatureFromAtom(terms[0], terms[1]);
+      if (foundFeature) {
+        features.push(foundFeature);
+      }
+    }
+    return features;
```

Comment:
Append the rest of features to the end of the list, if not found in findFeatureFromAtom